### ISSUE TITLE
animations: kinds of motion

### ISSUE THREAD

**Issue by kammitama5** [google/codeworld#604]
I tried to run it and expected the animation to run, but ```y``` is not in scope in the blue code snippet (ie it not declared explicitly).
I put in a Number value (it says y is of type Number) and it works just fine.

```
program  = animationOf(wheel)
wheel(t) = translated(rotated(tire, -60 * t), t - 10, y)
tire     = circle(1) & solidRectangle(0.1, 2)
```
![b_001](https://user-images.githubusercontent.com/8516378/39385399-4130f33c-4a25-11e8-9d5b-1fd47fccae72.png)

![b_001a](https://user-images.githubusercontent.com/8516378/39385401-44828b36-4a25-11e8-9bf7-8223ea989204.png)

**Comment by cdsmith** [comment]
Yes, that's a typo in https://github.com/google/codeworld/blob/master/web/help/GuideUnit3.md.  Feel free to send a pull request, or I'll fix it if you prefer.

**Comment by kammitama5** [comment]
I can just edit it directly.

**Comment by kammitama5** [comment]
Just did. Thank you!

**Comment by cdsmith** [comment]
Fixed by #605

**Comment by cdsmith** [comment]
Thanks a bunch.  Your change is merged and pushed to production.
